project/mission-vision.md

```diff
@@ -0,0 +1,14 @@
+## Mission
+
+To provide institutions a foundation to support the scientific data
+life-cycle.
+
+## Vision
+
+We will become a focal point for institutions to collaborate, share
+scientific data, integrate new technologies and develop their
+research portfolio. We will enable institutions to meet and exceed
+emerging sponsor data challenges, while maintaining the promises
+kept in the past. We will provide the platform to accellerate
+scientific research in future directions never thought possible
```

Comment:
We will provide the platform to accelerate scientific research in directions never before thought possible.

Reply:
Done.